src/components/molecules/GalleryGrid/GalleryGrid.jsx

```diff
@@ -0,0 +1,21 @@
+import gallery1 from "../../atoms/assets/images/gallery_img_1.png";
+import gallery2 from "../../atoms/assets/images/gallery_img_2.png";
+import gallery3 from "../../atoms/assets/images/gallery_img_3.png";
+import gallery4 from "../../atoms/assets/images/gallery_img_4.png";
+import gallery5 from "../../atoms/assets/images/gallery_img_5.png";
+import gallery6 from "../../atoms/assets/images/gallery_img_6.png";
+import "./GalleryGrid.scss";
+
+const GalleryGrid =()=> {
+    return (
+    <div className="gallery__images">
```

Comment:
use map function and map the data from JSON

Reply:
Done